### PR TITLE
Feature/memorial demo refactor

### PR DESCRIPTION
**Pull request by wijohnst**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "framer-motion": "^1.11.0",
+    "fs": "^0.0.1-security",
```
Comment (kklopfenstein) [inline package.json:10]: Is this needed?

File: pages/about.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import React from 'react'
```
Comment (kklopfenstein) [inline pages/about.js:1]: Are all these React imports needed?

**Review by kklopfenstein — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (wijohnst) [reply package.json:10]: Nope. I added these when I was trying to fix the FS problem.

File: pages/about.js (hunk 2 above)
Comment (wijohnst) [reply pages/about.js:1]: Not necessarily, but not having them was causing a lot of errors in VSCode. Possibly my Linting config is out of whack?

**Pull request merged**